cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.random().toString().slice(2, 6);
+  const username = faker.internet.userName() + '_' + randomNumber;
+  const email = `${username}@mail.com`;
+  const password = '12345Qwert!';
```

Comment:
The password '12345Qwert!' is hardcoded. It's a good practice to generate random and unique passwords for each user. You can use the `faker.internet.password()` method to generate a random password.